pkg/authorization/toplevel_org_authorizer.go

```diff
@@ -86,8 +86,8 @@ func (a *topLevelOrgAccessAuthorizer) Authorize(ctx context.Context, attr author
 		return a.delegate.Authorize(ctx, attr)
 	}
 
-	cluster, err := genericapirequest.ValidClusterFrom(ctx)
-	if err != nil || cluster == nil || cluster.Name.Empty() {
+	cluster := genericapirequest.ClusterFrom(ctx)
+	if cluster == nil || cluster.Name.Empty() {
 		kaudit.AddAuditAnnotations(
```

Comment:
this seems not symmetric with the other changes in the authorizer. I suggest to add audit log annotations there as well.

Reply:
okay, I think it only applies to the `LocalAuthorizer`, the other authorizers seem to be adding the annotation when the cluster wasn't found.